src/js/tech/tech.js

```diff
@@ -218,12 +218,13 @@ class Tech extends Component {
       return;
     }
 
+    let player = this.player();
     let textTracksChanges = function() {
       let updateDisplay = Fn.bind(this, function() {
-        this.trigger('texttrackchange');
+        player.trigger('texttrackchange');
```

Comment:
I was trying to track this down, but got confused between `texttrackchange` and `cuechange` events.

The Captions Settings control [explicitly calls an updateDisplay on the textTrackDisplay](https://github.com/videojs/video.js/blob/master/src/js/tracks/text-track-settings.js#L134) when a setting is changed. When a cue changes because of the video moving along the timeline, the track fires a `cuechange` event which [causes the tech to fire a `texttrackchange` event](https://github.com/videojs/video.js/blob/master/src/js/tech/tech.js#L221), which seems to just cause the [player to fire a `texttrackchange` event](https://github.com/videojs/video.js/blob/master/src/js/player.js#L930). Nothing fires an `updateDisplay` on the `textTrackDisplay`.